src/pages/Checkout2.page.js

```diff
@@ -0,0 +1,29 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+// eslint-disable-next-line import/prefer-default-export
+export class CheckoutPage2 extends BaseSwagLabPage {
```

Comment:
```suggestion
export class CheckoutStepTwoPage extends BaseSwagLabPage {
```
It looks better then just `CheckoutPage2` ( <- this looks like we just created one more, and dont give any info why this is 2)